### DISCUSSION TITLE
Welcome to snp500_stock_prediction Discussions!

### DISCUSSION THREAD

**Discussion by mltrev23** [mltrev23/snp500_stock_prediction#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by minion50000** [comment]
thank you for having me

**Reply by mltrev23** [reply]
hi

**Reply by mltrev23** [reply]
nice

**Comment by mltrev23** [comment]
welcome

**Comment by mltrev23** [comment]
welcome
